tests/unit/test_lambda.py

```diff
@@ -8,6 +8,7 @@
 
 from localstack import config
 from localstack.aws.accounts import get_aws_account_id
+from localstack.constants import TEST_AWS_ACCOUNT_ID, TEST_AWS_REGION_NAME
```

Comment:
Did @dfangl refer to these constants here: https://github.com/localstack/localstack/pull/9377#discussion_r1368390630 ?